tests/unit/aegra/test_auth_helpers.py

```
@@ -0,0 +1,50 @@
+            patch("deep_agent.aegra.auth.ENABLE_AUTH", True),
+            pytest.raises(HTTPException) as exc_info,
+        ):
+            await authenticated_user_id(request, reject_anonymous=True)
+        assert exc_info.value.status_code == 401
+
+    @pytest.mark.asyncio
+    async def test_extracts_sub_from_valid_jwt(self):
+        request = MagicMock()
+        request.headers = {"authorization": "Bearer valid-token"}
+        with (
+            patch("deep_agent.aegra.auth.ENABLE_AUTH", True),
```

Ruff (0.16.0)

[warning] Boolean positional value in function call

(FBT003)